core/src/components/datetime/test/basic/index.html

```diff
@@ -142,6 +142,18 @@
         </ion-item>
       </ion-list>
 
+      <ion-list>
+        <ion-item>
+          <ion-label>Display UTC 00:00 in Local Timezone (default behavior)</ion-label>
+          <ion-datetime display-format="MMM DD, YYYY HH:mm" value="2020-01-01T00:00:00Z"></ion-datetime>
+        </ion-item>
+
+        <ion-item>
+          <ion-label>Display UTC 00:00 in UTC (display-timezone = 'utc')</ion-label>
+          <ion-datetime display-format="MMM DD, YYYY HH:mm" value="2020-01-01T00:00:00Z" display-timezone="utc""></ion-datetime>
```

Comment:
Can we add another test that has something other than utc as well?

Reply:
done!